packages/dds/tree/src/test/domains/json/jsonCursor.spec.ts

```diff
@@ -3,7 +3,6 @@
  * Licensed under the MIT License.
  */
 
-// eslint-disable-next-line import/no-internal-modules
 import { singleJsonCursor, cursorToJsonObject } from "../../../domains/json/index.js";
```

Comment:
we don't want to allow cases like this without warnings. This is still reaching inside domains which should not be allowed by default.